python_src/models/police_report.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import annotations
+from typing import Tuple
+from abc import ABC, abstractmethod
+
+CAT_A = 'A'
+CAT_B = 'B'
+CAT_C = 'C'
+CAT_D = 'D'
+
+CAT_A_DATA = ["ANY"]
+CAT_B_DATA = ['F', 'M', 'B']
+CAT_C_DATA = ['L', 'R']
+CAT_D_DATA = ["FL", "FR", "ML", "MR", "BL", "BR"]
```

Comment:
In AC3R, i used "RL" and "RR" to denote a rear left and rear right damage. To avoid confusion of Vuong's successor, maybe we can use RL and RR instead of BL and BR?

Reply:
We use B for back, and R for Right